modules/pel/peltool/peltool.py

```diff
@@ -372,14 +400,12 @@ def main():
                         help='Delete original file after parsing')
     parser.add_argument('-t', '--termination', dest='critSysTerm',
                         action='store_true', help='List only critical system terminating PELs')
-    if inBMC:
-        parser.add_argument('-l', '--list',
-                            help='List PELs',
-                            action='store_true')
-    else:
-        parser.add_argument('-l', '--list',
-                            help='List PELs',
-                            dest='list')
+    parser.add_argument('-i', '--id',
+                        dest='pelID', help='Display a PEL based on its ID')
+    parser.add_argument('-P', '--path',
```

Comment:
As I mentioned elsewhere, let's swap -P and -p since the -p was just for debug and shouldn't have any real users.